From 95417bd47f80f232b7ebaedefad19c2bfdf46cd6 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Sun, 6 May 2018 11:25:06 +0300
Subject: [PATCH] Properly declare POD encoding
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This fixes the pod2man error

  YAWriter.pm around line 465: Non-ASCII character seen before =encoding in ''รถ''. Assuming CP1252
---
 YAWriter.pm | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/YAWriter.pm b/YAWriter.pm
index 38750e8..634e038 100644
--- a/YAWriter.pm
+++ b/YAWriter.pm
@@ -267,6 +267,8 @@ sub print {
 
 1;
 
+=encoding latin1
+
 =head1 NAME
 
 XML::Handler::YAWriter - Yet another Perl SAX XML Writer
-- 
2.17.0

